.circleci/config.yml

```diff
@@ -2112,7 +2126,11 @@ workflows:
           publish: true
           context:
             - oplabs-gcr
-            - slack
```

Comment:
Why remove `- slack`? I am unfamiliar with the slack details, but is this related to cross-building? Fine if it's cleanup

Reply:
This was a temp change so that I didn't produce a bunch of noisy slack notifications while testing the changes in this pr

Reply:
Added slack back in